spanner/spring-data/src/main/com/example/spanner/QuickStartSample.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import com.example.spanner.Singer.Name;
+import java.util.Arrays;
+
+/**
+ * A quick-start sample that uses Spring Data Cloud Spanner to perform read, write, and DDL
+ * operations.
+ */
+public class QuickStartSample {
+
+  public static void main(String[] args) throws InterruptedException {
+
+    SpannerSchemaToolsSample spannerSchemaToolsSample = new SpannerSchemaToolsSample();
+
+    /*
+      This call creates both the Singer and Album tables with an interleaved relationship.
+     */
+    spannerSchemaToolsSample.createTableIfNotExists();
+
+    Singer singer = new Singer();
+    singer.singerId = 1L;
+    singer.singerName = new Name("a first name", "a last name");
+    singer.albums = Arrays.asList(new Album(1L, 10L, "album1", 11L)
+        , new Album(1L, 20L, "album2", 12L));
+
+    /*
+      This call inserts the signer and performs a read query using a SpannerTemplate instance.
+     */
+    new SpannerTemplateSample().runTemplateExample(singer);
+
+    /*
+      This call uses queries defined in SingerRepository. The implementations of those queries
+      are generated by Spring Data Cloud Spanner.
+     */
+    new SpannerRepositorySample().runRepositoryExample();
+
+    /*
+      This call drops both the sample Singer and Album tables.
```

Comment:
Does it make sense to specify the method (that it uses the admin template)?

Reply:
Could you elaborate? I'm not understanding